web/src/lib/utils/licenseStatus.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Business,
+  getLicenseTasksProgress,
+  getNonLicenseTasks,
+  LicenseName,
+  Licenses,
+  LicenseStatusResults,
+  UserData,
+} from "@businessnjgovnavigator/shared/";
+
+export const getLicenseStatusResultsFromLicenses = (licenses: Licenses): LicenseStatusResults => {
```

Comment:
I created this function b/c the fn that created the task checkbox status expected the argument to pass in LicenseStatusResults. To avoid creating two functions that can determine the correct status, I created a function that modifies the shape so the same one can be used.